com.adsbynimbus.nimbus/Runtime/Scripts/NimbusManager.cs

```diff
@@ -485,6 +486,7 @@ private async Task<string> MakeRequestAsyncWithInterceptor(BidRequest bidRequest
 #endif
 		
 		private NimbusAdUnit RequestForNimbusAdUnit(BidRequest bidRequest, AdUnitType adUnitType) {
+			Debug.Log("BID REQUEST: " + JsonConvert.SerializeObject(bidRequest));
```

Comment:
This should be moved into whatever class hosts the method `MakeRequestAsyncWithInterceptor` and the log should use the result of the serialization operation in that method so the object is not serialized twice and if there is a failure in the serialization it will happen in a single spot making the debugging easier.